spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/index.adoc

```diff
@@ -15,7 +15,7 @@ Sabby Anandan, Artem Bilan, Marius Bogoevici, Eric Bottard, Mark Fisher, Ilayape
 :scst-starters-core-version: 1.0.2.RELEASE
 :sct-starters-core-version: 1.0.1.RELEASE
 
-:dataflow-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/spring-cloud-dataflow-docs/src/main/asciidoc
+:dataflow-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/v{scdf-core-version}/spring-cloud-dataflow-docs/src/main/asciidoc
```

Comment:
I wonder how this would work in case if we have `scdf-core-version` dependency as snapshot and we wouldn't have a tag for the snapshot.